site/manifest.json

```diff
@@ -96,6 +96,36 @@
       ],
       "is_security_patch": false
     },
+    {
+      "version": "1.11.8",
+      "flavor": "tetrate",
+      "flavor_version": 0,
+      "k8s_versions": [
+        "1.17",
+        "1.18",
+        "1.19",
+        "1.20"
+      ],
+      "release_notes": [
+        "https://istio.io/latest/news/releases/1.11.x/announcing-1.11.8/"
+      ],
+      "is_security_patch": false
+    },
+    {
+      "version": "1.11.8",
+      "flavor": "tetratefips",
+      "flavor_version": 0,
+      "k8s_versions": [
+        "1.17",
+        "1.18",
+        "1.19",
+        "1.20"
+      ],
+      "release_notes": [
+        "https://istio.io/latest/news/releases/1.11.x/announcing-1.11.8/"
+      ],
+      "is_security_patch": false
+    },
```

Comment:
Sorry, i missed this in earlier review, please add one more block like this for 1.11.8 with istio flavor.
``` 
    {
      "version": "1.13.2",
      "flavor": "istio",
      "flavor_version": 0,
      "k8s_versions": [
        "1.20",
        "1.21",
        "1.22",
        "1.23"
      ],
      "release_notes": [
        "https://istio.io/latest/news/releases/1.13.x/announcing-1.13.2/"
      ],
      "is_security_patch": false
    },
   ```

Reply:
I have added this. Just confirm if k8s version should be 1.17, 1.18, 1.19, 1.20 or 1.20, 1.21, 1.22, 1.23? Thanks.